src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -1364,28 +1364,29 @@ private IOperation FinishVisitingStatement(IOperation originalOperation, IOperat
             return result ?? MakeInvalidOperation(originalOperation.Syntax, originalOperation.Type, ImmutableArray<IOperation>.Empty);
         }
 
-        private void VisitStatements(ArrayBuilder<IOperation> statements)
-        {
-            foreach (var statement in statements)
-            {
-                VisitStatement(statement);
-            }
-        }
-
         private void VisitStatements(ImmutableArray<IOperation> statements)
         {
-            foreach (var statement in statements)
+            for (int i = 0; i < statements.Length; i++)
             {
-                VisitStatement(statement);
-            }
-        }
+                if (statements[i] is VariableDeclarationGroupOperation declarationGroup
+                    && declarationGroup.Declarations[0].DeclarationKind != DeclarationKind.Local)
```

Comment:
>&& declarationGroup.Declarations[0].DeclarationKind != DeclarationKind.Local [](start = 20, length = 76)

It would be good to not make assumption that all ```Declarations``` have the same declaration kind. Moving the property to the group should take care of that.